sources/light_client.move

```diff
@@ -510,14 +510,14 @@ public fun retarget_algorithm(p: &Params, previous_target: u256, first_timestamp
 /// * `proof`: merkle tree proof, this is the vector of 32bytes.
 /// * `tx_index`: index of transaction in block.
 /// * `transaction`: bitcoin transaction. Check transaction.move.
-/// * `receiver_address`: address of receiver in p2pkh or p2wpkh.
+/// * `receiver_pk_hash`: receiver public key hash in p2pkh or p2wpkh. Must not empty
 public fun verify_payment(
     lc: &LightClient,
     height: u64,
     proof: vector<vector<u8>>,
     tx_index: u64,
     transaction: &Transaction,
-    receiver_address: vector<u8>
+    receiver_pk_hash: vector<u8>
```

Comment:
much cleaner - I was wondering about it yesterday, because in the code I did it was PK hash, but in our SPV code just PK - that 's why I told @sczembor to verify with you.